include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,8 +1050,8 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+2), additional_data);
```

Comment:
why did you switch it back to `fe_degree+1`? Rebase errors? If so, `merge` is the answer :+1: Seriously.